eng/scripts/packagevalidation/ConfigValidation.csx

```diff
@@ -0,0 +1,15 @@
+#! "net5"
```

Comment:
It would be good to call the script what you expect it to return, such as IsOptimizedAssembly, well you would need to flip the bit to get that, but you get the idea name it after what it does.

Reply:
The returned integer becomes the exit code, so its better to keep it as it.